web/wp-content/themes/dctx/template-no-title.php

```diff
@@ -0,0 +1,29 @@
+<?php 
+
+/**
+* Template Name: No Title Page
+*
+* @package WordPress
+* @subpackage dctx
+* @since dctx 1.0
+*/
+	
+	get_header(); ?>
+
+
+<div id="main">
+	<div class="clearfix">
+
+		<div class="main-content">
+			<?php while (have_posts()) : the_post(); ?>
```

Comment:
:no_entry_sign: **Error**: No space after opening parenthesis is prohibited (*WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceAfterOpenParenthesis*).